storage/cloud-client/storage_compose_file.py

```diff
@@ -22,16 +22,25 @@
 
 def compose_file(bucket_name, sources, destination_blob_name):
     """Concatenate source blobs into destination blob."""
-    # bucket_name = "your-bucket-name"
-    # sources = [blob_1, blob_2]
```

Comment:
Instead I recommend doing:

Update compose_file() parameter list
```
    # bucket_name = "your-bucket-name"
    # blob1_name = "object-name-1"
    # blob2_name = "object-name-2"
    # destination_blob_name = "destination-object-name"
```

Then create the `sources` value in the sample as you did. 
This will require changes to the test file associated as well: 
https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/storage/cloud-client/snippets_test.py#L406-L420

Reply:
Updated. Thanks!